pkg/security/mcs/label.go

```diff
@@ -0,0 +1,178 @@
+package mcs
+
+import (
+	"bytes"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+const maxCategories = 1024
+
+type Labels struct {
```

Comment:
On 05/28/2015 03:40 PM, Paul Weil wrote:
>
> In pkg/security/mcs/label.go
> <https://github.com/openshift/origin/pull/2544#discussion_r31269852>:
>
> > @@ -0,0 +1,178 @@
> > +package mcs
> > +
> > +import (
> > +	"bytes"
> > +	"fmt"
> > +	"sort"
> > +	"strconv"
> > +	"strings"
> > +)
> > +
> > +const maxCategories = 1024
> > +
> > +type Labels struct {
>
> This leads me to believe that we should update |SecurityContext|
> upstream (post 3.0) to use better terminology too. The fields were
> largely based on how selinux items were passed via the docker run
> flags which uses "level" but encompasses level, category, and range.
>
> reference: https://docs.docker.com/reference/run/#security-configuration
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/2544/files#r31269852>.
>
I would leave them as level, this is the way most people refer to them
and I don't see anyone breaking them apart.